tools.sh

```diff
@@ -5,6 +5,7 @@ JQ=${JQ:-jq}
 QEMU_IMG=${QEMU_IMG:-qemu-img}
 VIRT_FILESYSTEMS=${VIRT_FILESYSTEMS:-virt-filesystems}
 GUESTFISH=${GUESTFISH:-guestfish}
```

Comment:
commit log says "virsh is part of different package", I'm not sure what this implies, different package from what?

Reply:
I still don't get the commit log. `libvirt-client` has been there since RHEL6, so "is now part" is kind of odd, this is not a recent change. As far as I can tell snc does not install any libvirt related packages, so isn't this change just making sure `virsh` is present before we try to use it?

Reply:
Also "and needs to be installed separately". But I still don't' fully get why the commit log is written tihs way :(